src/locales/en.json

```diff
@@ -52,7 +53,8 @@
     "createRoom": "Share a live-collaboration session",
     "duplicateSelection": "Duplicate",
     "untitled": "Untitled",
-    "name": "Name"
+    "name": "Name",
+    "madeInExcalidraw": "Made in Excalidraw"
```

Comment:
I'm not a native English speaker but this doesn't read well to me. Should it be `Made with Excalidraw` instead?

Reply:
Either makes sense to me. (in ≈ “inside of” vs with ≈ “using the Excalidraw tool”)

Reply:
Good point to bring up, I also like `Made with` (the iPad Pro ads used it too)